src/Containers/SearchFormContainer/SearchFormContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import { AddMovie } from '../AddMovie/AddMovie';
+import { SearchForm } from '../../Components/SearchForm/SearchForm';
+import { useSearchParams } from 'react-router-dom';
+
+export const SearchFormContainer = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  return (
+    <>
+      <SearchForm
+        initialSearchQuery={searchParams.get('search') ?? ''}
+        onSearch={(value) =>
+          setSearchParams((params) => {
+            params.set('search', value);
```

Comment:
According to the task description it should be `query`:

```suggestion
        initialSearchQuery={searchParams.get('query') ?? ''}
        onSearch={(value) =>
          setSearchParams((params) => {
            params.set('query', value);
```

Reply:
Also, it makes sense to move parameter names to a constant.